app/javascript/components/pages/EventIndex.test.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import EventIndex from './EventIndex'
+import EventIndex from './EventIndex';
 
-Enzyme.configure({ adapter: new Adapter() })
+Enzyme.configure({ adapter: new Adapter() });
+
```

Comment:
This test is great, but do you want to take another stab at getting it to test the component?